### DISCUSSION TITLE
Consul request failed with status [400]: Invalid service address

### DISCUSSION THREAD

**Discussion by 1450793561** [apache/skywalking#12325] [Q&A]
In SkyWalking cluster mode, using Consul as the configuration center, encountering an error upon starting according to the configuration shown in the screenshot. 

How to solve this problem, environmental variable configuration refers to https://skywalking.apache.org/docs/main/v9.7.0/en/setup/backend/configuration-vocabulary/ 

<img width="1658" alt="企业微信截图_3e18dc8d-48a7-4359-970d-1557b8bd1309" src="https://github.com/apache/skywalking/assets/31468018/ad39f78a-e303-401b-811c-413c932ee75b">
<img width="919" alt="企业微信截图_4f49cc70-8554-484b-8a33-ccc2be6da0d0" src="https://github.com/apache/skywalking/assets/31468018/817f2d78-8a55-44c6-85b9-f222aa287d9a">

**Comment by wu-sheng** [comment]
I think this is a typical Consul internal issue. Please refer to their docs and recheck the network address.

**Reply by 1450793561** [reply]
consul internal error? The oap service should not depend on consul. Is your oap service compatible with consul? I wonder how this kind of problem is solved?

**Reply by wu-sheng** [reply]
You are using consul as cluster coordinator. That is how this happens. We don't have expertise about Consul.
